### PR TITLE
kubvirt-vm-latnecy: Change VMs inteface binding method to bridge

### PR DESCRIPTION
**Pull request by ormergi**
Signed-off-by: Or Mergi <ormergi@redhat.com>

Depends on #75 and #73

### CODE REVIEW

File: checkups/kubevirt-vm-latency/vmlatency/internal/vmi/spec.go [hunk 1]
```diff
@@ -191,14 +191,23 @@ func WithSriovBinding() interfaceOption {
...
+			Bridge: &kvcorev1.InterfaceBridge{},
+		}
+	}
+}
+
 func NewAlpine(name string, opts ...Option) *kvcorev1.VirtualMachineInstance {
 	const (
-		memory                                     = "128Mi"
-		alpineContainerDiskImage                   = "quay.io/kubevirt/alpine-with-test-tooling-container-disk:v0.53.0"
+		memory                                     = "1Gi"
```
Comment (EdDev) [inline checkups/kubevirt-vm-latency/vmlatency/internal/vmi/spec.go:205]: I think we use 512Mi on kubevirt, no?
Please extract the image replacement to its own PR so we can take it in. It is a must for the MVP.

File: checkups/kubevirt-vm-latency/manifests/bridge-network-attachment-definition.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+      "cniVersion": "0.3.1",
+      "name": "br10",
+      "plugins": [
+          {
+              "type": "bridge",
+              "bridge": "br10"
+          },
+          {
+              "type": "tuning"
+          }
```
Comment (EdDev) [inline checkups/kubevirt-vm-latency/manifests/bridge-network-attachment-definition.yaml:18]: I am pretty sure this is no longer needed.
I should work without it.

File: checkups/kubevirt-vm-latency/manifests/bridge-network-attachment-definition.yaml [hunk 3]
```diff
@@ -0,0 +1,20 @@
+apiVersion: k8s.cni.cncf.io/v1
+kind: NetworkAttachmentDefinition
+metadata:
+  name: bridge-network
+  namespace: default
+spec:
+  config: |
+    {
+      "cniVersion": "0.3.1",
```
Comment (EdDev) [inline checkups/kubevirt-vm-latency/manifests/bridge-network-attachment-definition.yaml:9]: We can use a more recent one I guess.

Comment (EdDev) [reply checkups/kubevirt-vm-latency/manifests/bridge-network-attachment-definition.yaml:9]: not relevant

**Comment by ormergi** [comment]
Followed up at by #73
